api/lorawan.proto

```diff
@@ -107,26 +109,26 @@ enum DataRateIndex {
 }
 
 message MHDR {
-  MType m_type = 1;
-  Major major = 2;
+  MType m_type = 1 [(validate.rules).enum.defined_only = true];
+  Major major = 2 [(validate.rules).enum.defined_only = true];
 }
 
 message MACPayload {
   option (gogoproto.populate) = false;
 
-  FHDR f_hdr = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
-  uint32 f_port = 2 [(validator.field) = {int_lt: 256}];
-  bytes frm_payload = 3 [(gogoproto.customname) = "FRMPayload"];
+  FHDR f_hdr = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false, (validate.rules).message.required = true];
+  uint32 f_port = 2 [(validate.rules).uint32.lte = 255];
+  bytes frm_payload = 3 [(gogoproto.customname) = "FRMPayload", (validate.rules).bytes.max_len = 249];
```

Comment:
Remove upper bound

Reply:
Why?
This cannot be bigger by LoRaWAN spec